templates/multi_x_science_sum/templates.yaml

```diff
@@ -0,0 +1,52 @@
+dataset: multi_x_science_sum
+templates:
+  2bca0197-e3d4-4870-bd95-178411e52e09: !Template
+    id: 2bca0197-e3d4-4870-bd95-178411e52e09
+    jinja: 'Use the reference abstracts to generate related work:
+
+
+      {{ref_abstract.values()}}  |||
+
+      {{related_work}}'
+    name: ref_relatedwork
+    reference: ''
+  3bd082cb-4e28-4eb7-9fa2-dd03f1f86219: !Template
+    id: 3bd082cb-4e28-4eb7-9fa2-dd03f1f86219
+    jinja: 'Given the abstract of a paper, provide some related work for readers to
+      learn further
+
+      {{abstract}} |||
+
+      {{related_work}}'
+    name: abstract_relatedwork
+    reference: ''
+  af4d550e-54b8-471e-97af-2b2c50a1382e: !Template
+    id: af4d550e-54b8-471e-97af-2b2c50a1382e
+    jinja: 'Use the related work to guess the abstract:
+
+
+      {{related_work}} |||
+
+      {{abstract}}'
+    name: relatedwork_abstract
+    reference: ''
+  b9eb67b4-415b-4a38-a90a-9dee3ae385d7: !Template
+    id: b9eb67b4-415b-4a38-a90a-9dee3ae385d7
+    jinja: 'Use the abstract and reference abstracts to generate related work:
+
+
+      {{abstract}} , {{ref_abstract.values()}}  |||
+
+      {{related_work}}'
```

Comment:
Does including the reference abstracts make it too easy to predict the related work? Also, does abstract + ref_abstract.values() result in a really long input sequence?

Reply:
I feel that this task does not seem too easy as the reference abstracts, though they might convey the similar/same information as related work, are phrased in different words to that of related work. So the model would be challenged to understand the meaning at some level. I took a look at the prompt dataset viewer and they do not look too long for the most part